Android/toonDra/.gitignore

```diff
@@ -0,0 +1,46 @@
+# Built application files
+*.apk
+*.ap_
+
+# Files for the ART/Dalvik VM
+*.dex
+
+# Java class files
+*.class
+
+# Generated files
+bin/
+gen/
+out/
+
+# Gradle files
+.gradle/
+build/
+
+# Local configuration file (sdk path, etc)
+local.properties
+
+# Proguard folder generated by Eclipse
+proguard/
+
+# Log Files
+*.log
+
+# Android Studio Navigation editor temp files
+.navigation/
+
+# Android Studio captures folder
+captures/
+
+# Intellij
+*.iml
+.idea/
+
+# Keystore files
+*.jks
+
+# OSX
+.DS_Store
+
+# etc
+!gradle-wrapper.jar
```

Comment:
This has been already done in commit 0f3915b.
Remove changes for this file (you can choose not to include this file for commit).
Also it is a good practice to always submit a pull request with your commit rebased to latest master commit.
`git rebase master`